sv_pipeline/exome/load_data.py

```diff
@@ -198,7 +227,9 @@ def parse_sv_row(row, parsed_svs_by_id, header_indices, sample_id):
         parsed_svs_by_id[variant_id][GENOTYPES_FIELD] = []
         parsed_svs_by_id[variant_id][GENES_FIELD] = set()
 
-    sample_info = get_parsed_column_values(row, header_indices, SAMPLE_COLUMNS)
+    sample_info = get_parsed_column_values(row, header_indices, SAMPLE_COLUMNS) if is_new_joint_call else \
+        get_parsed_column_values(row, header_indices, MERGED_CALL_SAMPLE_COLUMNS, col_configs=MERGED_CALL_COL_CONFIGS)
```

Comment:
this is pretty hard to read/ maintain, calling the same function with half the args different like this. I think it might be clearer if you made both `columns` and `col_configs` optional in `get_parsed_column_values`.  If `col_configs` is passed in it loads all of those columns, otherwise it uses the subset of columns from the `columns` args. Then this call would be
```
kwargs = {'columns': SAMPLE_COLUMNS} if is_new_joint_call else {'col_configs': MERGED_CALL_COL_CONFIGS}
sample_info = get_parsed_column_values(row, header_indices, **kwargs) 
```

Reply:
We don't have a sample column config yet. If we load all columns in the col_configs, it will include more columns than the sample ones.